roomranger/frontend/src/components/roomAttendants/AddAttendant.js

```diff
@@ -5,175 +5,223 @@ import AttendantListComponent from "./AttendantListComponent";
+                    name="phoneNumber"
+                    value={phoneNumber}
+                    onChange={(e) => handleChange(e)}
+                  ></input>
+                </div>
+              </div>
+
+
+
+              <div className="row">
+                <div className=" col-md-6 mb-3">
+                  <label htmlFor="Username" className="form-label">
+                    {/* Username */}
+                  </label>
+                  <input
+                    type={"text"}
+                    className="form-control"
+                    placeholder="Username"
+                    name="username"
+                    value={username}
+                    onChange={(e) => handleChange(e)}
+                  ></input>
+                </div>
+                <div className="col-md-6">
+                  <label htmlFor="Password" className="form-label">
+                    {/* Password */}
+                  </label>
+                  <input
+                    type={"password"}
+                    className="form-control"
+                    placeholder="Password"
+                    name="password"
+                    //value={password}
+                    onChange={(e) => handleChange(e)}
+                  ></input>
+                </div>
+              </div>
             <div className="row">
-          {/* <div>
-            <label htmlFor="Pronoun" className="form-label col-md-1"></label>
-            <select name="pronoun" value={pronoun} onChange={(e) => onInputChange(e)} >
-                <option value="Choose" >Pronoun</option>
-                <option value="He/Him" >He/Him</option>
-                <option value="She/Her" >She/Her</option>
-                <option value="They/Them" >They/Them</option>
-            </select> */}
+              <div className="col-md-6 mb-3">
+                  <label htmlFor="Password" className="form-label">
+                    {/* Confirm Password */}
+                  </label>
+                  <input
+                    type={"password"}
+                    className="form-control"
+                    placeholder="Verify Password"
+                    name="password"
+                    // value={password}
+                    onChange={(e) => handleChange(e)}
+                  ></input>
+             </div>
+               
+                <div className="col-md-6 mb-4">
+                  <label htmlFor="notes" className="form-label ">
+                    {/* NOTES */}
+                  </label>
+                  <textarea class="form-control h-100 " id="notes" name="notes" placeholder="Notes"  onChange={(e) => handleChange(e)}></textarea>
+                </div>
+                </div >
+                <div className="container" >
+                  <label  htmlFor="workingDays" className="form-label col-md-1 checkbox-inline"><h5>Working Days</h5></label>
+                  {['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'].map((day) => (
```

Comment:
Monday to Sunday?

Reply:
Yes, you are right. I will fix this and add to the PR. Thanks.